docs/facts/examples.ipynb

```diff
@@ -279,6 +279,31 @@
    "id": "3b1b4c39",
    "metadata": {},
    "outputs": [
+    {
+     "name": "stdout",
```

Comment:
I think you might need to rerun the book command. Run `make` and then commit the chnages.